hstream/admin.go

```diff
@@ -9,7 +9,17 @@ type Stats struct {
 	Values []float64
 }
 
-func (c *HStreamClient) AdminRequest(cmd string) (string, error) {
+// AdminRequestToRandomServer send admin command request to a random server in the cluster
+func (c *HStreamClient) AdminRequestToRandomServer(cmd string) (string, error) {
```

Comment:
let `AdminRequest` send req to random server by default, and another method `AdminRequestToServer` send req to a specific server. Otherwise, you should modify all the places that use `AdminRequest` before to `AdminRequestToRandomServer`

Reply:
done